techniques/css/C37.html

```diff
@@ -2,86 +2,66 @@
 <html lang="ja">
    <head>
       <meta charset="UTF-8" />
-      <title>C37: Using CSS max-width and height to fit images</title>
+      <title>C37: 画像を適応させるために、CSS の max-width と height を使用する</title>
```

Comment:
fitを「収める」と取るのはどうでしょう？

```suggestion
      <title>C37: 画像を収めるために、CSS の max-width 及び height を使用する</title>
```